css/components/button.css

```diff
@@ -0,0 +1,124 @@
+.btn{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    min-width: 6rem;
+    height: 3rem;
+    padding: 0 var(--space-sm);
+    font-weight: bold;
+    background-color: var(--gray-opaque);
+    border: 2px solid var(--gray-opaque);
+    border-radius: var(--space-xs);
+    transition: all 0.3s ease-in-out;
+}
+
+.btn:hover{
+    border-color: var(--gray-hover);
+    background-color: var(--gray-hover);
+}
+
+.btn-primary{
+    border-color: var(--primary-light);
+    background-color: var(--primary-light);
+    color: var(--background-day);
```

Comment:
You can give color to the btn class. It is repeated in 2-3 class.

Reply:
> You can give color to the btn class. It is repeated in 2-3 class.

Hi thanks for the heads up. It  had totally gone unnoticed :D thanks. I have refactored the code.